index.html

```diff
@@ -0,0 +1,189 @@
+    <span>Glad to see you!</span>
+    </div>
+        <div class="bio">
+         <p>I’m a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+         </div>
+        <div class="lets_contact">
+          <div class="contact">
+            <h2>LET’S CONNECT</h2>
+        </div>
+       
+            <section class="social_media">
+                <ul class="social-icons-set">
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-twitter" target="_blank" aria-label="Twitter"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-linkedin-in" target="_blank" aria-label="Linkedin"></i></a>
+                  </li>
+                  <li class="social-icon"><a href="#"><i class="fa-solid fa-m" target="_blank"
+                        aria-label="Medium"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-github" target="_blank" aria-label="GitHub"></i></a>
+                  </li>
+                  <li class="social-icon">
+                    <a href="#"><i class="fa-brands fa-angellist" target="_blank" aria-label="Angellist"></i></a>
+                  </li>
+                </ul>
+              </section>
+            </section>
+          </section>
+    </div>
+    <section class="middle_section">
+        <div class="pic">
+    
+        </div>
+        <div class="name">
+            <h1>
+                Mahram Hossain
+            </h1>
+        </div>
+        <div class="descript">
+            <div class="camp">
+                <h5>
+                    CANOPY
+                </h5>
+            </div>
+            <div class="year">
+                <ul class="list">
+                    <li>Back And Dev</li>
+                    <li>2015</li>
+                </ul>
+            </div>
+        </div>
+        <div class="task">
+            <p>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+        </div>
+        <div class="web">
+            <div class="right">
+                <ul>
+                    <li>
+                        <a href="#">HTML</a>
+                    </li>
+                    <li>
+                        <a href="#">CSS</a>
+                    </li>
+                    <li>
+                        <a href="#">JavaScript</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="left"></div>
+        </div>
+        <div class="link">
+            <a href="#">
+                See project
+            </a>
+        </div>
+    </section>
+    <section class="about">
+        <div class="am">
+            <h1>
```

Comment:
- [ ] please use a different hx tag here. your page having just one h1 tag helps for semantic purposes
- [ ] kindly use the correct text color for about my self #172B4D;